static/js/util/util_test.js

```diff
@@ -217,4 +220,57 @@ describe('utility functions', () => {
       });
     });
   });
+
+  describe('profile sort functions', () => {
+    it('should sort by resume order', () => {
+      let entries = ['1969', '1997', '1992', '1934'].map(year => (
+        { 'date': moment(year, 'YYYY') }
```

Comment:
We shouldn't pass in a moment object here since it's not how we're using `resumeOrder` elsewhere. Can you format this moment object to ISO-8601?